Source/Scene/SceneTransitioner.js

```diff
@@ -1,50 +1,50 @@
 /*global define*/
 define([
-        '../Core/Cartesian3',
-        '../Core/Cartographic',
-        '../Core/defined',
-        '../Core/destroyObject',
-        '../Core/DeveloperError',
-        '../Core/EasingFunction',
-        '../Core/Math',
-        '../Core/Matrix4',
-        '../Core/Ray',
-        '../Core/ScreenSpaceEventHandler',
-        '../Core/ScreenSpaceEventType',
-        '../Core/Transforms',
-        './Camera',
-        './OrthographicFrustum',
-        './OrthographicOffCenterFrustum',
-        './PerspectiveFrustum',
-        './SceneMode'
-    ], function(
-        Cartesian3,
-        Cartographic,
-        defined,
-        destroyObject,
-        DeveloperError,
-        EasingFunction,
-        CesiumMath,
-        Matrix4,
-        Ray,
-        ScreenSpaceEventHandler,
-        ScreenSpaceEventType,
-        Transforms,
-        Camera,
-        OrthographicFrustum,
-        OrthographicOffCenterFrustum,
-        PerspectiveFrustum,
-        SceneMode) {
+    '../Core/Check',
+    '../Core/Cartesian3',
+    '../Core/Cartographic',
+    '../Core/defined',
+    '../Core/destroyObject',
+    '../Core/DeveloperError',
+    '../Core/EasingFunction',
+    '../Core/Math',
+    '../Core/Matrix4',
+    '../Core/Ray',
+    '../Core/ScreenSpaceEventHandler',
+    '../Core/ScreenSpaceEventType',
+    '../Core/Transforms',
+    './Camera',
+    './OrthographicFrustum',
+    './OrthographicOffCenterFrustum',
+    './PerspectiveFrustum',
+    './SceneMode'
+], function(
+    Check,
+    Cartesian3,
+    Cartographic,
+    defined,
+    destroyObject,
+    DeveloperError,
+    EasingFunction,
+    CesiumMath,
+    Matrix4,
+    Ray,
+    ScreenSpaceEventHandler,
+    ScreenSpaceEventType,
+    Transforms,
+    Camera,
+    OrthographicFrustum,
+    OrthographicOffCenterFrustum,
+    PerspectiveFrustum,
+    SceneMode) {
```

Comment:
Looks like this whole block got de-dented by two spaces. Could indent it back to where it was? It will also make the `DeveloperError` change more clear. Thanks!